src/testers.jl

```diff
@@ -164,6 +168,7 @@ end
  - If `check_inferred=true`, then the inferrability (type-stability) of the `rrule` is checked
    — if `f` is itself inferrable — along with the inferrability of the pullback it returns.
  - `fkwargs` are passed to `f` as keyword arguments.
+ - `testset_name`: if provided, the name of the testset used to wrap the tests.
```

Comment:
```suggestion
- `testset_name`: if provided, the name of the testset used to wrap the tests.
  Otherwise it is determined from the function and argument types.
```